src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/DRConfig/RemoveAzureServiceBusDRConfiguration.cs

```diff
@@ -0,0 +1,56 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+namespace Microsoft.Azure.Commands.ServiceBus.Commands.GeoDR
+{
+    /// <summary>
+    /// 'Remove-AzureRmServicebusDRConfigurations' Cmdlet Deletes an Alias(Disaster Recovery configuration)
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, ServicebusDRConfigurationVerb, SupportsShouldProcess = true)]
+    public class RemoveServicBusDRConfiguration : AzureServiceBusCmdletBase
```

Comment:
@v-Ajnava you will need to implement two additional parameter sets for this cmdlet:

1) A set with just an `InputObject` parameter of type `ServiceBusDRConfigurationAttributes` that accepts its value from the pipeline
2) A set with just a `ResourceId` parameter of type `string` that accepts its value from the pipeline by property name.

This will enable to piping scenarios that are outlined in the [Azure PowerShell Piping Scenarios](https://github.com/Azure/azure-powershell/wiki/Piping-in-PowerShell#azure-powershell-piping-scenarios) section of our piping document. Please let me know if you have any questions about this.

Reply:
As Alias is not a Tracked resource, we are implementing on the InputObject parameter sets.